dsl/src/test/scala/com/crobox/clickhouse/dsl/language/CollectionFunctions.scala

```diff
@@ -22,7 +22,7 @@ class CollectionFunctions extends ColumnFunctionTest {
 
   it should "succeed for IN functions" in {
     val someCollection = Seq(1,4,6,9)
-    val someTuple = tuple(1,4,6,9)
+    val someTuple = tuple(1: Int, 4: Int, 6: Int, 9: Int)
```

Comment:
I don't think we want to have those type hints, are they required to make it compile?

Reply:
Yes. Note that only type hints make it compile in 2.13.

Reply:
Ah, I Found what the issue was that doesn't require any changes to the test. This must be a requirement since you don't want todo that in all you implementation code. Opened PR #97 that adds scala 2.13 support.